argon2-jvm-nolibs/src/main/java/de/mkammerer/argon2/jna/Argon2Library.java

```diff
@@ -168,6 +165,84 @@ ARGON2_PUBLIC int argon2_hash(const uint32_t t_cost, const uint32_t m_cost,
      */
     int argon2id_verify(byte[] encoded, byte[] pwd, Size_t pwdlen);
 
+    /*
+    ARGON2_PUBLIC int argon2_verify(const char *encoded, const void *pwd,
+                                const size_t pwdlen, argon2_type type);
+     */
+    int argon2_verify(byte[] encoded, byte[] pwd, Size_t pwdlen, Argon2_type type);
+
+    /**
+     * Argon2d: Version of Argon2 that picks memory blocks depending
```

Comment:
The method is named `argon2i`, but the comment says `argon2d`.

Reply:
Sorry, that happened when sorting the methods to match the structure of the file.